src/app/api/v1/addTestimonial/route.ts

```diff
@@ -0,0 +1,64 @@
+import jwt from "jsonwebtoken";
+import { NextRequest, NextResponse } from "next/server";
+import Testimonial from "../../../../model/Testimonial";
+import dbConnect from "../../../../lib/dbConnect";
+
+interface TestimonialInter {
+  userId: number;
+  content: string;
+  createdAt: Date;
+}
+
+const JWT_SECRET = process.env.JWT_TOKEN_SECRET;
+const testimonials: TestimonialInter[] = [];
+
+export async function POST(req: NextRequest) {
+  await dbConnect();
+  try {
+    const { content } = await req.json();
```

Comment:
The request method check is placed after calling req.json(). This is inefficient and could lead to unintended behavior